tensorflow/compiler/tf2tensorrt/kernels/trt_engine_op.cc

```diff
@@ -289,6 +292,8 @@ TRTEngineOp::TRTEngineOp(OpKernelConstruction* context)
   }
   OP_REQUIRES_OK(context, context->GetAttr("max_cached_engines_count",
                                            &max_cached_engines_));
+  OP_REQUIRES_OK(context,
```

Comment:
As mentioned in ops/trt_engine_op.cc, please use implicit attribute and don't use `OP_REQUIRES_OK`, but set it to `true` if the attribute is not set.

Reply:
Done